frontend/src/container/ListOfDashboard/RequestDashboardBtn.tsx

```diff
@@ -0,0 +1,95 @@
+import '../../pages/Integrations/Integrations.styles.scss';
```

Comment:
We should move the shared styles to a common file and use the common file in this case.

Reply:
Since the UI will effect other 3 places I'm not scoping the optimization to our scope. We are going to change this UI all together.